scripts/lib/install_node.sh

```diff
@@ -0,0 +1,21 @@
+# Install Node and necessary frameworks/packages
+
+# Install Node.js on Arch Linux
+install_node() {
+  command -v node >/dev/null 2>&1 || {
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash
+    source ~/.nvm.sh
```

Comment:
Do you mean ~/.nvm/nvm.sh ?

Reply:
I don't think so. I ran the script and it was able to complete its task just fine. This was to allow nvm to be used immediately after installing without the need to close and reopen the terminal.

Reply:
Hmmm ok I think the reason why I didn't encounter this was because when I tested it again, the nvm source string was already in my /.bashrc file. Thanks for pointing this out.